src/amo/components/AddonDetail.js

```diff
@@ -90,6 +91,13 @@ class AddonDetail extends React.Component {
           <div dangerouslySetInnerHTML={sanitizeHTML(nl2br(addon.description),
                                                      allowedDescriptionTags)} />
         </section>
+
+        <hr />
```

Comment:
ah, right. The mock has them so I guess we could use `border-bottom` or something?